include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1032,93 +1101,113 @@ class FEPointEvaluation
    * precomputes the @p shapes vector, holding the evaluation of 1D basis
    * functions of tensor product polynomials, if necessary.
    */
+  template <bool is_face, bool is_linear>
   void
   do_reinit();
 
   /**
    * Resizes necessary data fields, reads in and renumbers solution values.
    * Interpolates onto face if face path is selected.
    */
+  template <bool is_face_path, bool is_linear, size_t stride_view>
   void
   prepare_evaluate_fast(
-    const ArrayView<const ScalarNumber>    &solution_values,
-    const EvaluationFlags::EvaluationFlags &evaluation_flags);
+    const StridedArrayView<const ScalarNumber, stride_view> &solution_values,
+    const EvaluationFlags::EvaluationFlags                  &evaluation_flags);
 
   /**
    * Evaluates the actual interpolation on the cell or face for a quadrature
    * batch.
    */
+  template <bool is_face_path, bool is_linear, size_t stride_view>
```

Comment:
Can we uniformly use 
```suggestion
  template <bool is_face_path, bool is_linear, std::size_t stride_view>
```